.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -54,6 +57,36 @@ body:
     attributes:
       label: Other information
       placeholder: Screenshots, additional logs, notes, etc.
+  - type: markdown
+    attributes:
+      value: |
+        ---
+        ## Additional context
+
+        Please provide whatever additional information you have available to you. If not, please scroll to the bottom and submit the issue.
+  - type: dropdown
+    id: browser
+    attributes:
+      label: Browser
+      description: (You may select more than one)
+      options:
+        - Google Chrome/Chromium
+        - Mozilla Firefox
+        - Microsoft Edge
+        - Apple Safari
+        - iOS Safari
+        - Android Chrome
+      multiple: true
+  - type: dropdown
+    id: site-type
+    attributes:
+      label: Platform (Simple, Atomic, or both?)
```

Comment:
That felt a bit odd to me at first as well, but this is how it's done on the Calypso side today:
- https://github.com/Automattic/wp-calypso/blob/trunk/.github/ISSUE_TEMPLATE/bug_report.yml
- https://github.com/Automattic/wp-calypso/blob/trunk/.github/ISSUE_TEMPLATE/happiness-bug-report.yml

I included it that way to be consistent since we're expecting more issues reported directly into the Jetpack repo from HEs in the future.

If we left it out of this general `bug-report.yml` template, I think we should create a new template, just like Calypso does, for Happiness reports. That seems a bit overkill to me, especially since the Atomic / Simple verbiage is still used in the general bug report template there.

What do you think?

Reply:
> How would a wpcom customer tell if they're on Simple or Atomic? Do they see "Simple" or "Atomic" somewhere? Or something else? Or would they have to figure it out based on their plan or something?

They wouldn't be able to know. "Simple" and "Atomic" are internal codenames, so I do not think any non-automattician would be able to fill this in.

Since the platform doesn't necessarily match your plan, we cannot use plan names here either.

Arguably, non-automatticians will not be able to fill in issue severity info either. What if I were to move the existing horizontal line higher, and add a mention that the fields below are only meant to be filled by Automatticians or something along those lines?

<img width="1246" alt="Screen Shot 2022-07-12 at 08 02 25" src="https://user-images.githubusercontent.com/426388/178419930-600c43be-01d9-42ff-8366-e45a0f172302.png">

Reply:
That could work. I note people could reasonably fill in the reproducibility and workaround details though.

And you know they're probably going to try to guess at the severity anyway. Possibly exaggerating to get their issue more attention. 😀

Reply:
Good enough